packages/venia-ui/lib/components/ProductImageCarousel/carousel.js

```diff
@@ -16,6 +16,16 @@ import Image from '../Image';
 const DEFAULT_IMAGE_WIDTH = 640;
 const DEFAULT_IMAGE_HEIGHT = 800;
 
+/**
+ * Carousel component for product images
```

Comment:
Can you provide a more detailed description for people not familiar with an image carousel?

Reply:
@jcalcaben added more description on carousel 👍